include/clad/Differentiator/BuiltinDerivatives.h

```diff
@@ -82,6 +82,55 @@ ValueAndPushforward<int, int> cudaDeviceSynchronize_pushforward()
     __attribute__((host)) {
   return {cudaDeviceSynchronize(), 0};
 }
+
+template <typename T>
+__global__ void atomicAdd_kernel(T* destPtr, T* srcPtr, size_t N) {
+  for (int i = blockIdx.x * blockDim.x + threadIdx.x; i < N;
```

Comment:
```suggestion
  for (size_t i = blockIdx.x * blockDim.x + threadIdx.x; i < N;
```

`int i` and `size_t N` have different types, thus different ranges. You can use `auto` as well.